bigtable-hbase-1.x-parent/bigtable-hbase-1.x-mapreduce/src/main/java/com/google/cloud/bigtable/mapreduce/validation/BigtableSyncTableJob.java

```diff
@@ -384,6 +433,31 @@ private Job setupJobWithBigtable(String[] otherArgs) throws IOException {
       jobConf.set(HConstants.ZOOKEEPER_ZNODE_PARENT, zkClusterKey.getZnodeParent());
     }
 
+    // set bigtable target configuration
+    if (targetBigtableProjectId != null && targetBigtableInstance != null) {
+      if (targetZkCluster != null)
+        LOG.warn(
+            "targetZkCluster config("
+                + targetZkCluster
+                + ") overridden with targetBigtableProjectId("
+                + targetBigtableProjectId
+                + "), targetBigtableInstance("
+                + targetBigtableInstance
+                + ")");
+
+      jobConf.set(TARGET_BT_PROJECTID_CONF_KEY, targetBigtableProjectId);
+      jobConf.set(TARGET_BT_INSTANCE_CONF_KEY, targetBigtableInstance);
+
+      if (targetBigtableAppProfile != null)
+        jobConf.set(TARGET_BT_APP_PROFILE_CONF_KEY, targetBigtableAppProfile);
```

Comment:
please add curly brackets and scope the if statement, everywhere.

Reply:
updated